docs/ca/dev/opentelemetry/how-to-instrument.md

```diff
@@ -0,0 +1,428 @@
+---
+title: OpenTelemetry overview
+description: Describing what is OpenTelemetry and how Spryker integrates it in the SCCOS
+template: howto-guide-template
+last_updated: Feb 1, 2025
+related:
+---
+
+You may want to get familiar with [OpenTelemetry Documentation](https://opentelemetry.io/docs/concepts/) first, but we are going to give you a brief overview here too.
+
+
+### Convention following
+The current implementation follows [OpenTelemetry Semantic Conventions 1.30.0](https://opentelemetry.io/docs/specs/semconv/).
```

Comment:
implementation of what?

Reply:
Implementation of Spryker Monitoring Integration